dummy-data.js

```diff
@@ -1,92 +1,92 @@
 const dummyData = [
   {
-    id: "a",
     username: "philzcoffee",
     thumbnailUrl:
       "https://tk-assets.lambdaschool.com/ecd33d34-c124-4b75-92d2-e5c52c171ed8_11201517_887808411287357_1307163552_a.jpg",
 
     imageUrl:
```

Comment:
don't know if I would necessarily change the data that was given to me for a project. You wouldn't necessarily be able to do that in the real world. Try to work with what you're given. If you need to add a boolean for liked...do it in your state and toggle it there.